composer.json

```diff
@@ -3,7 +3,7 @@
     "require": {
         "php": "^8.1",
         "cron/cron": "^1.8.0",
-        "ext-redis": "^3.1"
+        "ext-redis": "*"
```

Comment:
I prefer restrictive versions, why to allow them all here ?

Reply:
This is a feature, or a lack of feature, of our PHP buildpack. Related issue: https://github.com/Scalingo/php-buildpack/issues/395